### PR TITLE
OCPBUGS-38891 - 4.14 RN for known issue - network outages

### PR DESCRIPTION
**Pull request by slovern**
OCPBUGS-38891 - 4.14 RN for known issue - network outages

Version(s): 4.14

Issue: https://issues.redhat.com/browse/OCPBUGS-38891

Link to docs preview: [Known issues](https://81411--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-14-release-notes.html#ocp-4-14-known-issues)

QE review:
- [x] QE has approved this change.

Additional information:

This change requires **Change Management**.

Change Management Approvals:
- [x] Product Experience: Edu Alcaniz @ealcaniz
- [x] QE: Zhanqi Zhao
- [x] Engineering:  Carlos Goncalves @cgoncalves
[Carlos Gonçalves](https://github.com/cgoncalves) 
- [x] Engineering PM: Marc Curry @mcurry-rh 
- [x] DPM: Sheldon @tradej
- [x] Content Strategist: Donagh Brennan @DonaghBR 
- [x] Release Notes Stakeholder: Darragh Fitzmaurice @dfitzmau

### CODE REVIEW

**Comment by andreaskaris** [comment]
for what it's worth, lgtm

**Comment by zhaozhanqi** [comment]
I saw the bug https://issues.redhat.com/browse/OCPBUGS-36710 is for LGW, right? if yes, Do we need to highlight this.  cc @anuragthehatter

**Comment by anuragthehatter** [comment]
> I saw the bug https://issues.redhat.com/browse/OCPBUGS-36710 is for LGW, right? if yes, Do we need to highlight this. cc @anuragthehatter

Thats right @zhaozhanqi

**Comment by andreaskaris** [comment]
It affected me only on local gateway mode (well, that's what I tested with). I'm not sure if it's more generalized. However, please go ahead and mention that it's only with LGW / routingViaHost: true
Thanks for catching this

**Comment by slovern** [comment]
> I saw the bug https://issues.redhat.com/browse/OCPBUGS-36710 is for LGW, right? if yes, Do we need to highlight this. cc @anuragthehatter

Thanks @zhaozhanqi I've updated the text to mention LGW. If it looks ok to you can you give it an lgtm?

**Comment by zhaozhanqi** [comment]
LGTM,  cc @anuragthehatter to confirm

File: release_notes/ocp-4-14-release-notes.adoc [hunk 1]
```diff
@@ -3321,6 +3321,10 @@ Contact Red Hat Support for additional details and assistance.
 
 * There is a known issue in this release preventing access to the Web Terminal Operator after it has been installed. This issue will be fixed in a future {product-title} release. (link:https://issues.redhat.com/browse/OCPBUGS-14463[*OCPBUGS-14463*])
 
+* There is a known issue when upgrading the Cluster Network Operator from version 4.13 to 4.14.
```
Comment (lahinson) [inline release_notes/ocp-4-14-release-notes.adoc:3324]:
```suggestion
* A known issue exists when you upgrade the Cluster Network Operator from version 4.13 to 4.14.
```
Per IBM Style, avoid starting a sentence with the phrase "There is" or "There are".

File: release_notes/ocp-4-14-release-notes.adoc [hunk 2]
```diff
@@ -3321,6 +3321,10 @@ Contact Red Hat Support for additional details and assistance.
...
+The conversion to the new OVN Kubernetes interconnect multi-zone architecture can lead to packet drops that cause network outages of up to 300ms per node. 
+The impact is on East/West traffic in local gateway mode with `routingViaHost` set to `true`.
```
Comment (lahinson) [inline release_notes/ocp-4-14-release-notes.adoc:3326]:
```suggestion
The impact is on East/West traffic in local gateway mode with `routingViaHost` set to `true`. (link:https://issues.redhat.com/browse/OCPBUGS-38891[*OCPBUGS-38891*])
```

**Review by lahinson — COMMENTED** [review]
@slovern I believe this update requires Change Management according to the OpenShift docs manual: https://docs.google.com/document/d/1fLMpK4bqthtFlCwA36yeo2cI-pExFpX76VMnSi_In6Q/edit#heading=h.nckpao8upx1u

File: release_notes/ocp-4-14-release-notes.adoc (hunk 1 above)
Comment (slovern) [reply release_notes/ocp-4-14-release-notes.adoc:3324]: I was following the pattern here:
https://redhat-documentation.github.io/supplementary-style-guide/#_known_issue
Does the supplementary guide take precedence over IBM or the other way around?

File: release_notes/ocp-4-14-release-notes.adoc (hunk 2 above)
Comment (slovern) [reply release_notes/ocp-4-14-release-notes.adoc]: Thanks for the review @lahinson 
I've included the link as you suggested

**Comment by ealcaniz** [comment]
LGTM

**Comment by cgoncalves** [comment]
LGTM from a technical/engineering perspective.

**Comment by slovern** [comment]
> LGTM from a technical/engineering perspective.

Thanks for reviewing @cgoncalves

**Comment by sub-mod** [comment]
LGTM

**Comment by mcurry-rh** [comment]
LGTM !

**Comment by tradej** [comment]
LGTM

**Comment by dfitzmau** [comment]
LGTM. Merging.

**Pull request merged**